{
    "type": "bundle",
    "id": "bundle--613eb7ae-e110-41ea-bbe6-9ab0cfd3c730",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--ef2ec95d-d1fd-4729-9bc7-250f4631ad2b",
            "created_by_ref": "identity--748e6444-f073-4c50-b558-f49be8897a81",
            "created": "2022-06-15T17:15:00.000Z",
            "modified": "2022-11-11T19:15:00.000Z",
            "name": "CVE-2022-32156",
            "description": "In Splunk Enterprise and Universal Forwarder versions before 9.0, the Splunk command-line interface (CLI) did not validate TLS certificates while connecting to a remote Splunk platform instance by default. After updating to version 9.0, see Configure TLS host name validation for the Splunk CLI https://docs.splunk.com/Documentation/Splunk/9.0.0/Security/EnableTLSCertHostnameValidation#Configure_TLS_host_name_validation_for_the_Splunk_CLI to enable the remediation. The vulnerability does not affect the Splunk Cloud Platform. At the time of publishing, we have no evidence of exploitation of this vulnerability by external parties. The issue requires conditions beyond the control of a potential bad actor such as a machine-in-the-middle attack. Hence, Splunk rates the complexity of the attack as High.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2022-32156"
                },
                {
                    "source_name": "cwe",
                    "external_id": "CWE-295"
                },
                {
                    "source_name": "https://www.splunk.com/en_us/product-security/announcements/svd-2022-0606.html",
                    "url": "https://www.splunk.com/en_us/product-security/announcements/svd-2022-0606.html"
                },
                {
                    "source_name": "https://docs.splunk.com/Documentation/Splunk/9.0.0/Security/Updates",
                    "url": "https://docs.splunk.com/Documentation/Splunk/9.0.0/Security/Updates"
                },
                {
                    "source_name": "https://docs.splunk.com/Documentation/Splunk/9.0.0/Security/EnableTLSCertHostnameValidation#Configure_TLS_host_name_validation_for_the_Splunk_CLI",
                    "url": "https://docs.splunk.com/Documentation/Splunk/9.0.0/Security/EnableTLSCertHostnameValidation#Configure_TLS_host_name_validation_for_the_Splunk_CLI"
                },
                {
                    "source_name": "cve2stix",
                    "description": "This object was created using cve2stix from the Signals Corps.",
                    "url": "https://github.com/signalscorps/cve2stix"
                }
            ],
            "object_marking_refs": [
                "marking-definition--613f2e26-407d-48c7-9eca-b8e91df99dc9"
            ],
            "extensions": {
                "extension-definition--b2b5f2cd-49e6-4091-a0e0-c0bb71543e23": {
                    "extension_type": "property-extension",
                    "nvd_cve": {
                        "data_type": "CVE",
                        "data_format": "MITRE",
                        "data_version": "4.0",
                        "CVE_data_meta": {
                            "ID": "CVE-2022-32156",
                            "ASSIGNER": "prodsec@splunk.com"
                        },
                        "problemtype": {
                            "problemtype_data": [
                                {
                                    "description": [
                                        {
                                            "lang": "en",
                                            "value": "CWE-295"
                                        }
                                    ]
                                }
                            ]
                        },
                        "references": {
                            "reference_data": [
                                {
                                    "url": "https://www.splunk.com/en_us/product-security/announcements/svd-2022-0606.html",
                                    "name": "https://www.splunk.com/en_us/product-security/announcements/svd-2022-0606.html",
                                    "refsource": "CONFIRM",
                                    "tags": [
                                        "Vendor Advisory"
                                    ]
                                },
                                {
                                    "url": "https://docs.splunk.com/Documentation/Splunk/9.0.0/Security/Updates",
                                    "name": "https://docs.splunk.com/Documentation/Splunk/9.0.0/Security/Updates",
                                    "refsource": "CONFIRM",
                                    "tags": [
                                        "Release Notes",
                                        "Vendor Advisory"
                                    ]
                                },
                                {
                                    "url": "https://docs.splunk.com/Documentation/Splunk/9.0.0/Security/EnableTLSCertHostnameValidation#Configure_TLS_host_name_validation_for_the_Splunk_CLI",
                                    "name": "https://docs.splunk.com/Documentation/Splunk/9.0.0/Security/EnableTLSCertHostnameValidation#Configure_TLS_host_name_validation_for_the_Splunk_CLI",
                                    "refsource": "CONFIRM",
                                    "tags": [
                                        "Mitigation",
                                        "Vendor Advisory"
                                    ]
                                }
                            ]
                        },
                        "description": {
                            "description_data": [
                                {
                                    "lang": "en",
                                    "value": "In Splunk Enterprise and Universal Forwarder versions before 9.0, the Splunk command-line interface (CLI) did not validate TLS certificates while connecting to a remote Splunk platform instance by default. After updating to version 9.0, see Configure TLS host name validation for the Splunk CLI https://docs.splunk.com/Documentation/Splunk/9.0.0/Security/EnableTLSCertHostnameValidation#Configure_TLS_host_name_validation_for_the_Splunk_CLI to enable the remediation. The vulnerability does not affect the Splunk Cloud Platform. At the time of publishing, we have no evidence of exploitation of this vulnerability by external parties. The issue requires conditions beyond the control of a potential bad actor such as a machine-in-the-middle attack. Hence, Splunk rates the complexity of the attack as High."
                                }
                            ]
                        }
                    },
                    "impact": {
                        "baseMetricV3": {
                            "cvssV3": {
                                "version": "3.1",
                                "vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
                                "attackVector": "NETWORK",
                                "attackComplexity": "LOW",
                                "privilegesRequired": "NONE",
                                "userInteraction": "NONE",
                                "scope": "UNCHANGED",
                                "confidentialityImpact": "HIGH",
                                "integrityImpact": "HIGH",
                                "availabilityImpact": "HIGH",
                                "baseScore": 9.8,
                                "baseSeverity": "CRITICAL"
                            },
                            "exploitabilityScore": 3.9,
                            "impactScore": 5.9
                        },
                        "baseMetricV2": {
                            "cvssV2": {
                                "version": "2.0",
                                "vectorString": "AV:N/AC:M/Au:N/C:P/I:P/A:P",
                                "accessVector": "NETWORK",
                                "accessComplexity": "MEDIUM",
                                "authentication": "NONE",
                                "confidentialityImpact": "PARTIAL",
                                "integrityImpact": "PARTIAL",
                                "availabilityImpact": "PARTIAL",
                                "baseScore": 6.8
                            },
                            "severity": "MEDIUM",
                            "exploitabilityScore": 8.6,
                            "impactScore": 6.4,
                            "acInsufInfo": false,
                            "obtainAllPrivilege": false,
                            "obtainUserPrivilege": false,
                            "obtainOtherPrivilege": false,
                            "userInteractionRequired": false
                        }
                    },
                    "publishedDate": "2022-06-15T17:15Z",
                    "lastModifiedDate": "2022-11-11T19:15Z"
                }
            }
        },
        {
            "type": "indicator",
            "spec_version": "2.1",
            "id": "indicator--b60f902a-0a9e-4dbe-b9ec-400968fcae01",
            "created_by_ref": "identity--748e6444-f073-4c50-b558-f49be8897a81",
            "created": "2022-06-15T17:15:00.000Z",
            "modified": "2022-11-11T19:15:00.000Z",
            "name": "CVE-2022-32156",
            "description": "In Splunk Enterprise and Universal Forwarder versions before 9.0, the Splunk command-line interface (CLI) did not validate TLS certificates while connecting to a remote Splunk platform instance by default. After updating to version 9.0, see Configure TLS host name validation for the Splunk CLI https://docs.splunk.com/Documentation/Splunk/9.0.0/Security/EnableTLSCertHostnameValidation#Configure_TLS_host_name_validation_for_the_Splunk_CLI to enable the remediation. The vulnerability does not affect the Splunk Cloud Platform. At the time of publishing, we have no evidence of exploitation of this vulnerability by external parties. The issue requires conditions beyond the control of a potential bad actor such as a machine-in-the-middle attack. Hence, Splunk rates the complexity of the attack as High.",
            "indicator_types": [
                "compromised"
            ],
            "pattern": "[(software:cpe='cpe:2.3:a:splunk:splunk:*:*:*:*:enterprise:*:*:*' OR software:cpe='cpe:2.3:a:splunk:universal_forwarder:*:*:*:*:*:*:*:*')]",
            "pattern_type": "stix",
            "pattern_version": "2.1",
            "valid_from": "2022-06-15T17:15:00Z",
            "external_references": [
                {
                    "source_name": "cve2stix",
                    "description": "This object was created using cve2stix from the Signals Corps.",
                    "url": "https://github.com/signalscorps/cve2stix"
                }
            ],
            "object_marking_refs": [
                "marking-definition--613f2e26-407d-48c7-9eca-b8e91df99dc9"
            ],
            "extensions": {
                "extension-definition--b463c449-d022-48b7-b464-3e9c7ec5cf16": {
                    "extension_type": "property-extension",
                    "all_cpe23uris": [
                        "cpe:2.3:a:splunk:splunk:*:*:*:*:enterprise:*:*:*",
                        "cpe:2.3:a:splunk:universal_forwarder:*:*:*:*:*:*:*:*"
                    ],
                    "vulnerable_cpeid": [
                        "cpe:2.3:a:splunk:splunk:*:*:*:*:enterprise:*:*:*",
                        "cpe:2.3:a:splunk:universal_forwarder:*:*:*:*:*:*:*:*"
                    ]
                }
            }
        },
        {
            "type": "relationship",
            "spec_version": "2.1",
            "id": "relationship--98611cab-f2da-447e-9008-43fb4a1516e7",
            "created_by_ref": "identity--748e6444-f073-4c50-b558-f49be8897a81",
            "created": "2022-06-15T17:15:00.000Z",
            "modified": "2022-11-11T19:15:00.000Z",
            "relationship_type": "identifies",
            "source_ref": "indicator--b60f902a-0a9e-4dbe-b9ec-400968fcae01",
            "target_ref": "vulnerability--ef2ec95d-d1fd-4729-9bc7-250f4631ad2b",
            "external_references": [
                {
                    "source_name": "cve2stix",
                    "description": "This object was created using cve2stix from the Signals Corps.",
                    "url": "https://github.com/signalscorps/cve2stix"
                }
            ]
        }
    ]
}